src/Avalonia.FuncUI.Elmish/Avalonia.FuncUI.Elmish.fsproj

```diff
@@ -15,15 +15,17 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="ElmishHook.fs" />
     <Compile Include="Library.fs" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Elmish" Version="3.1.0" />
-    <PackageReference Update="FSharp.Core" Version="4.7.2" />
+    <PackageReference Update="FSharp.Core" Version="6.0.1" />
```

Comment:
Not sure if this was automatic, but last time while I was upgrading Avalonia to the latest version you asked me to keep 4.7.2 for better backwards compatibility, so we should probably do the same in this project as well 😃

Reply:
Haha that is true! 

@JaggerJo requested that I make the Elmish project depend on the new Components project. 
The new Components project was net5.0 / fsharp 6.0.1 (and was actually using some new stuff - I think in the form of a Readonly attribute), while all the other projects were netstandard2.0. So the only way to do it was to upgrade the Elmish project.

So a decision needs to be made on whether to stay behind or move ahead to new stuff.
But it looks like maybe that decision has already been made. Thoughts @JaggerJo?

Reply:
I'm completely up to upgrade all the FSharp.Core packages to the latest version, so sounds good to me 😃